R/Sound_DMR.R

```diff
@@ -812,24 +812,31 @@ add_changepoint_info <- function(whole_df, changepoint, col_name) {
   MethRegionName <- paste0('MethRegion_', col_name)
   MethRegionLengthName <- paste0('MethRegionLength_', col_name)
   MethGroupName <- paste0('MethGroup_', col_name)
+  MethCytosinesName <- paste0('MethCytosines_', col_name)
   
   if (typeof(changepoint) != 'NULL') {
     changepoint <<- changepoint
     changepoints <- changepoint@cpts
     
     #Add in the first and last cytosine if needed for the changepoint start and stop sites
     if ((1 %in% changepoints) & (nrow(whole_df) %in% changepoints)) {
```

Comment:
One day I might try to simplify this logic, but I remember why I had to write an if statement to account for every possibility